src/TripSegment/index.js.flow

```diff
@@ -0,0 +1,25 @@
+// @flow
+import List from "../List";
+
+export type TripType = "airline" | "bus" | "train";
+
+export type State = {|
+  shown: boolean,
+|};
+
+export type Props = {|
+  +children: React$Element<typeof List>,
+  +code: string,
+  +name: string,
+  +arrivalTime: string,
+  +arrivalCity: string,
```

Comment:
Can be `arrivalCity` and `arrivalCode` merge into one prop? Same for `departureCity` and `departureCode`